hawc/apps/lit/management/commands/topic_model.py

```diff
@@ -0,0 +1,33 @@
+from django.core.management.base import BaseCommand
```

Comment:
scope creep, but added this mgmt commands so I could easily test to make sure the sklearn changes actually worked as expected.